src/components/Messanger/MessengeForm.jsx

```diff
@@ -0,0 +1,78 @@
+import React, {Component} from 'react';
+import PropType from 'prop-types';
+import {Box, Button, Card, FormControl, TextField} from '@material-ui/core';
+
+
+export class MessengeForm extends Component {
+    state = {
+        author: '',
+        text: '',
+        created_at: Date()
+    };
+
+    static propTypes = {
+        onSend: PropType.func,
+    };
+
+    handleMessageSend = () => {
+        const {onSend} = this.props;
+
+        if (typeof onSend !== 'function') {
+            return false;
+        }
+
+        onSend(this.state);
+
+        this.setState({
+            text: '',
+            author: '',
+        });
+    };
+
+    handleInputChange = ({target}) => {
+        let fieldName = target.name;
+
+        this.setState({
+            [fieldName]: target.value
+        });
+    };
+
+    render() {
+
+        const {author, text} = this.state;
+
+        return (
+
+            <FormControl className={'message__form'}> <Card className={'message__form-card'}>
```

Comment:
рекомендую определиться со своим стилем :) и писать одинаково везде
```suggestion
            <FormControl className="message__form"> <Card className="message__form-card">
```